src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/dnn-persona-bar-page-treeview/src/PersonaBarPageTreeview.jsx

```diff
@@ -133,44 +126,25 @@ export class PersonaBarPageTreeview extends Component {
             draggedItem,
             Localization } = this.props;
 
-        const hotspotStyles = {
-            position: "relative",
-            zIndex: 9997,
-            wordWrap: "break-word",
-            textOverflow: "wrap",
-            width: "100%",
-            height: "20px",
-            marginTop: "-20px"
-            //backgroundColor: "rgb(0,1,2,.5)"
-        };
-
         let index = 0;
         let total = listItems.length;
         return listItems.map((item) => {
             const name = this.trimName(item);
-            const shouldShowTooltip = /\.\.\./.test(name);
             const canManagePage = (e, item, fn) => {
                 const message = Localization.get("NoPermissionManagePage");
-                const left = () => {
-                    e ? fn(e, item) : fn(item);
-                };
-                const right = () => {
-                    this.props.setEmptyPageMessage(message);
-                };
+                const left = () => e ? fn(e, item) : fn(item);
+                const right = () => this.props.setEmptyPageMessage(message);
```

Comment:
I was thinking to maybe start calling these in relation to the actual condition check? In here we could have some thing like `item.canManagePage ? whenCanManage() : whenCannotManage()`.

Something to maintain but maybe still better the what you'd get after reversing the condition 😂 

```
item.canManagePage === false ? right() : left();
```